### DISCUSSION TITLE
Is it Possible ? i want to make a website ,,,,,,, ,user data will push continously like a web hosting ?

### DISCUSSION THREAD

**Discussion by Mehedi-Hasan-soft-web-support** [community/community#146597] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it Possible ? i want to make a website ,,,,,,, ,user data will push continously like a web hosting ?

**Comment by shinybrightstar** [comment]
Hi @Mehedi-Hasan-soft-web-support , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.
